### PR TITLE
SONIC CLI for CLI-Sessions feature

### PR DESCRIPTION
**Pull request by i-davydenko**
HLD: https://github.com/sonic-net/SONiC/pull/1367

| Module name  | PR  | state | context |
| ------------- | ------------- | ----|-----|
| [sonic-buildimage](https://github.com/sonic-net/sonic-buildimage) | [Dev cli sessions](https://github.com/sonic-net/sonic-buildimage/pull/17623) | GitHub issue/pull request detail | GitHub pull request check contexts |
| [sonic-host-services](https://github.com/sonic-net/sonic-host-services)  | [cli-sessions](https://github.com/sonic-net/sonic-host-services/pull/99) | GitHub issue/pull request detail | GitHub pull request check contexts |
| [sonic-utilities](https://github.com/sonic-net/sonic-utilities)  | [SONIC CLI for CLI-Sessions feature #3175](https://github.com/sonic-net/sonic-utilities/pull/3175) | GitHub issue/pull request detail   | GitHub pull request check contexts  |

#### What I did

Implement next commands for CLI-sessions feature:

- config serial-console inactivity-timeout  
- config serial-console sysrq-capabilities
- show serial-console

- config ssh  max-sessions
- config ssh  inactivity-timeout 
- show ssh

#### How I did it
Write handlers in config/main.py for serial-console and ssh commands to cover configuration set;
Write handlers in show/main.py for serial-console and ssh to cover show commands.
#### How to verify it
Manual tests

#### Previous command output (if the output of a command-line utility has changed)
n/a
#### New command output (if the output of a command-line utility has changed)
n/a

### CODE REVIEW

File: config/main.py [hunk 1]
```diff
@@ -7372,5 +7372,69 @@ def date(date, time):
...
+    """Set ssh inactivity timeout"""
+
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    config_db.mod_entry("SSH_SERVER", 'POLICIES',
+                        {'inactivity_timeout': inactivity_timeout})
+
+@ssh.command('max-sessions')
+@click.argument('max-sessions', metavar='<max-sessions>', required=True, 
+                type=click.IntRange(0,100))
```
Comment (qiluo-msft) [inline config/main.py:7429]: >100 [](http://example.com/codeflow?start=38&length=3)

Why picking this value? I could not see it in yang model.

Comment (i-davydenko) [reply config/main.py:7429]: @qiluo-msft upper limit for this config is specified in HLD and in yang models, please take a look:
https://github.com/sonic-net/sonic-buildimage/pull/17623/files#diff-7202f7348dad4b5be36f441f7ff0a5d6468395cbf89a9b75aa87399cef07292dR2598

**Review by qiluo-msft — DISMISSED** [review]

**Comment by qiluo-msft** [comment]
Please resolve conflicts.

**Comment by fastiuk** [comment]
@qiluo-msft , you approved all the rest. Could you please approve that one?

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**